README.md

```diff
@@ -1 +1,36 @@
-# clojure-tic-tac-toe
\ No newline at end of file
+# tic-tac-toe
+
+FIXME: description
```

Comment:
I would remove or upload the generic placeholder info

Reply:
Done